bin/create_cluster_config.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/python
```

Comment:
We should use Python 3, and it looks like this script is, so the shebang should be `#!/usr/bin/env python3`

Reply:
yes, agree

Reply:
this was fixed